dashboard/modules/job/pydantic_models.py

```diff
@@ -37,9 +37,6 @@ class JobDetails(BaseModel):
     """
 
     type: JobType = Field(..., description="The type of job.")
-    entrypoint: Optional[str] = Field(
-        None, description="The entrypoint command for this job."
-    )
```

Comment:
Why remove this?

Reply:
it's duplicate with https://github.com/ray-project/ray/pull/35124/files/e231a5ca7141af307f3cefd16cbb31e48789f134#diff-d1972d6e703f3aa016325518da046bfa3647810e6315e8bd351f95912a02d8a8L66

Reply:
Ah good catch